url-shortener/src/main/java/com/bobocode/shortener/controller/URLShortenerController.java

```diff
@@ -0,0 +1,36 @@
+package com.bobocode.shortener.controller;
+
+import com.bobocode.shortener.dto.URLPayload;
+import com.bobocode.shortener.service.URLShortenService;
+import com.bobocode.shortener.utils.UrlUtils;
+import lombok.RequiredArgsConstructor;
+import org.springframework.http.HttpHeaders;
+import org.springframework.http.HttpStatus;
+import org.springframework.http.ResponseEntity;
+import org.springframework.web.bind.annotation.*;
+
+@RestController
+@RequestMapping("/api")
+@RequiredArgsConstructor
+public class URLShortenerController {
+
+    private final URLShortenService urlShortenService;
+
+    @PostMapping("/short/urls")
+    public ResponseEntity<?> shortenUrl(@RequestBody final URLPayload payload) {
+        final String shortenUrl = this.urlShortenService.shortenUrl(payload);
+        return ResponseEntity
+                .status(HttpStatus.CREATED)
+                .header(HttpHeaders.LOCATION, shortenUrl)
```

Comment:
@vromantsev `shortenUrl` is this case is something like `boboshort.3435`, but the **header `Location` must contain a complete location** (complete URL).

Reply:
Here is the requirement from the task:

Reply:
@vromantsev "shortened URL" means shortened URL, not the id :)

In other words, this is an url: 
* https://github.com/bobocode-projects/java-fundamentals-exercises/tree/main/3-0-java-core/3-6-4-random-field-comparator

and this is a shortened URL
* https://shorturl.at/goq37